Actions/BuildALDoc/BuildALDoc.ps1

```diff
@@ -0,0 +1,108 @@
+﻿Param(
+    [Parameter(HelpMessage = "The GitHub token running the action", Mandatory = $false)]
+    [string] $token,
+    [Parameter(HelpMessage = "Artifacts to build documentation for", Mandatory = $true)]
+    [string] $artifacts
+)
+
+. (Join-Path -Path $PSScriptRoot -ChildPath "..\AL-Go-Helper.ps1" -Resolve)
+. (Join-Path -Path $PSScriptRoot -ChildPath "BuildALDoc.HelperFunctions.ps1" -Resolve)
+DownloadAndImportBcContainerHelper
+
+$settings = $env:Settings | ConvertFrom-Json
+$projects = $settings.ALDoc.Projects
+$excludeProjects = $settings.ALDoc.ExcludeProjects
+$maxReleases = $settings.ALDoc.MaxReleases
+$artifactsFolder = Join-Path $ENV:GITHUB_WORKSPACE ".artifacts"
+$artifactsFolderCreated = $false
+if ($artifacts -eq ".artifacts") {
```

Comment:
It looks like the only value for`$artifacts` is `.artifacts`. If it's not that, it doesn't matter what value it is.

In CICD it's called with `.artifacts` and in the separate workflow, as `latest`. That's inconsistent and could probably be improved.